webapp/app/viewModels/role/roleList.js

```diff
@@ -0,0 +1,24 @@
+require('services/roleService');
+
+define(['services/roleService'], function (roleService) {
+  var vm = {
+    getView: function () {
+      var view = require('./roleList.html');
+      return $.parseHTML(view)[0];
+    },
+    roles: ko.observableArray([])   
+    };
+    
+
+    vm.activate = function () {
+
+        return roleService.roleReadAll().then(function (response) {
+            console.log(response.ProcessJsonRequestsPostResult[0].Roles);
```

Comment:
When you finish you debug task remove this line

Reply:
i remove the console.log